packages/camera/camera_android_camerax/pubspec.yaml

```diff
@@ -2,7 +2,8 @@ name: camera_android_camerax
 description: Android implementation of the camera plugin using the CameraX library.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_android_camerax
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-publish_to: 'none'
+
+version: 0.5.0
 
 environment:
   sdk: ">=2.18.0 <4.0.0"
```

Comment:
Has this ever actually been run with 3.3? If not you may want to start it at 3.7.